pkg/apis/aadpodidentity/v1/types.go

```diff
@@ -166,48 +166,48 @@ const (
 // and the identities present..
 type AzureIdentityBindingSpec struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	AzureIdentity     string `json:"azureidentity"`
-	Selector          string `json:"selector"`
+	AzureIdentity     string `json:"AzureIdentity"`
+	Selector          string `json:"Selector"`
 	// Weight is used to figure out which of the matching identities would be selected.
-	Weight int `json:"weight"`
+	Weight int `json:"Weight"`
 }
 
 type AzureIdentityBindingStatus struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	AvailableReplicas int32 `json:"availableReplicas"`
+	AvailableReplicas int32 `json:"AvailableReplicas"`
 }
 
 /*** AzureAssignedIdentitySpec ***/
 
 //AzureAssignedIdentitySpec has the contents of Azure identity<->POD
 type AzureAssignedIdentitySpec struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	AzureIdentityRef  *AzureIdentity        `json:"azureidentityref"`
-	AzureBindingRef   *AzureIdentityBinding `json:"azurebindingref"`
-	Pod               string                `json:"pod"`
-	PodNamespace      string                `json:"podnamespace"`
-	NodeName          string                `json:"nodename"`
+	AzureIdentityRef  *AzureIdentity        `json:"AzureIdentityRef"`
+	AzureBindingRef   *AzureIdentityBinding `json:"AzureBindingRef"`
+	Pod               string                `json:"Pod"`
+	PodNamespace      string                `json:"PodNamespace"`
+	NodeName          string                `json:"NodeName"`
 
-	Replicas *int32 `json:"replicas"`
+	Replicas *int32 `json:"Replicas"`
 }
 
 // AzureAssignedIdentityStatus has the replica status of the resouce.
 type AzureAssignedIdentityStatus struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	Status            string `json:"status"`
-	AvailableReplicas int32  `json:"availableReplicas"`
+	AvailableReplicas int32  `json:"AvailableReplicas"`
 }
 
 // AzurePodIdentityExceptionSpec matches pods with the selector defined.
 // If request originates from a pod that matches the selector, nmi will
 // proxy the request and send response back without any validation.
 type AzurePodIdentityExceptionSpec struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	PodLabels         map[string]string `json:"podLabels"`
+	PodLabels         map[string]string `json:"PodLabels"`
 }
 
 // AzurePodIdentityExceptionStatus ...
 type AzurePodIdentityExceptionStatus struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Status            string `json:"status"`
+	Status            string `json:"Status"`
```

Comment:
why are we changing this status field to `Status`? Isn't it similar to what other CRDs have defined?